src/blocks/temperature.rs

```diff
@@ -128,13 +169,15 @@ impl Block for Temperature {
                 self.text.set_text(self.output.clone());
             }
 
-            self.text.set_state(match max {
-                0...20 => State::Good,
-                21...45 => State::Idle,
-                46...60 => State::Info,
-                61...80 => State::Warning,
-                _ => State::Critical,
-            });
+            let state =
```

Comment:
I'd suggest moving back to the `match` syntax here. It's a little more verbose in this case but easier to reason about.

Reply:
Done in dd0cd52